docs/build/index.md

```diff
@@ -66,6 +66,9 @@ Ethereum has a large and growing number of tools to help developers build, test,
 
 ## Integrated Development Environments (IDEs)
 
+### Play *Web based lightweight Solidity editor with a live smart contract preview*
```

Comment:
play.ethereum.org feels incomplete to me right now. 
I wouldn't put it at the top of this list.

Reply:
> This page will never be completely comprehensive: the goal is to list the most popular or widely used resources or tools.